docs/config/modules/opentsdb.xml

```diff
@@ -78,14 +78,28 @@
   <section>
     <title>Examples</title>
     <example>
-      <title>Ingesting OpenTSDB data</title>
+      <title>Ingesting opentsdb data</title>
+      <para>
+        <code>
+          &lt;Plugin write_tsdb&gt;
+            &lt;Node&gt;
+              Host           "localhost"
+              Port           "4242"
+              HostTags       "datacenter=east hosttype=backend"
+              StoreRates     false
+              AlwaysAppendDS false
+            &lt;/Node&gt;
+          &lt;/Plugin&gt;
+        </code>
+      </para>
+      <para>The collectd write_tsdb plugin config example above will direct metric data to the broker:4242 for ingestion</para>
```

Comment:
Should be "opentsdb" rather than "collectd"?

Reply:
yes. a bug for @vasuraman11